.pytool/Plugin/DscCompleteCheck/DscCompleteCheck.py

```diff
@@ -58,7 +59,7 @@ def RunBuildPlugin(self, packagename, Edk2pathObj, pkgconfig, environment, PLM,
                 "DscPath not found in config file.  Nothing to check.")
             return -1
 
-        abs_pkg_path = Edk2pathObj.GetAbsolutePathOnThisSystemFromEdk2RelativePath( # MU_CHANGE
+        abs_pkg_path = Edk2pathObj.GetAbsolutePathOnThisSystemFromEdk2RelativePath(  # MU_CHANGE

```

Comment:
As far as I can tell, you just moved the comment here. Was that to align it on a multiple of 4 column?

Reply:
That was probably my formatter adhering to [PEP8](https://peps.python.org/pep-0008/#inline-comments): "Inline comments should be separated by at least two spaces from the statement."